packages/cloud-config/definition.yaml

```diff
@@ -1,3 +1,3 @@
 name: cloud-config
 category: system
-version: 0.9-7
+version: 0.9-8
```

Comment:
0.10 or 0.0.10 or 0.10.0 ???

Reply:
> 0.10 or 0.0.10 or 0.10.0 ???

Right.. we could go `x.y.z`, but as it is a cloud-init bundle I wouldn't go much complex. In the future makes sense to even split this package in small sub-packages with a meta (so just pieces of it can be installed), so I'd keep the versioning simple